lang_java/parsing/tree-sitter-parser.js

```diff
@@ -0,0 +1,62 @@
+const Parser = require('tree-sitter');
+const Java = require('../parsing/tree-sitter-java'); 
+
+var myArgs = process.argv.slice(2);
```

Comment:
You might wanna use some script best practices like usage documentation and number of parameter handling. I think https://github.com/tree-sitter/tree-sitter-cli/blob/master/cli.js is a decent example for this

Reply:
Thanks! I'll add it in a future iteration.